portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/InstanceSettings.testcase

```diff
@@ -1394,6 +1394,7 @@ definition {
 	@description = "This is a use case for LPS-156105. The test assert the value of mail setting for new instance is inherited from server level."
 	@priority = 2
 	test ViewInstanceMailSettingsValueIsInheritedFromServerLevel {
+		property portal.upstream = "quarantine";
```

Comment:
@joshitagaki This is under "Instance Setting" component, need to notify Platform Experience Team that you are adding this test to quarantine. Their team will need to modify their test suite to catch quarantine